src/styles/blocks/page.scss

```diff
@@ -0,0 +1,12 @@
+.page {
+  min-width: 1024px;
```

Comment:
You can try to add max-width with the formula
`margin: 0 auto;`
`max-width: ($card-width*$card-per-row)
    +($hor-padding*2)+($gap*$card-per-row)-$gap;`